src/ring_vrf_impl.rs

```diff
@@ -277,10 +295,14 @@ impl GenerateVerifiable for BandersnatchVrfVerifiable {
 		member: &Self::Member,
 	) -> bool {
 		use ark_vrf::ietf::Verifier;
-		let signature = IetfVrfSignature::deserialize_compressed(signature.as_slice()).unwrap();
+		let Ok(signature) = IetfVrfSignature::deserialize_compressed(signature.as_slice()) else {
+			return false;
+		};
```

Comment:
This I don't know in which circumstance it actually panics, but panicking doesn't look good. Better to say the signature is invalid.

Is there other part of the code where we eagerly panic?

Reply:
it actually panics if signature is `[0, 0..]`